atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKvsSerializableTransactionWithCellLockTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.keyvalue.cassandra;
+
+import static org.mockito.Mockito.mock;
+
+import org.junit.ClassRule;
+
+import com.palantir.atlasdb.containers.CassandraContainer;
+import com.palantir.atlasdb.containers.Containers;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.sweep.queue.MultiTableSweepQueueWriter;
+import com.palantir.atlasdb.sweep.queue.SpecialTimestampsSupplier;
+import com.palantir.atlasdb.sweep.queue.TargetedSweepFollower;
+import com.palantir.atlasdb.sweep.queue.TargetedSweeper;
+import com.palantir.atlasdb.transaction.impl.AbstractSerializableTransactionWithCellLockTest;
+import com.palantir.lock.v2.TimelockService;
+
+public class CassandraKvsSerializableTransactionWithCellLockTest
```

Comment:
Note: you're definitely only doing this because of precedence, so i'm cool with you leaving it - but I'll ask anyway as food for thought - is there really a behaviour that having a Cassandra around will catch issues with that your unit tests won't?

Reply:
right now you're just running all the tests 3 times, but it's not like they even use the KVS particularly